lita_config.rb.example

```diff
@@ -19,4 +19,32 @@ Lita.configure do |config|
 
   config.robot.adapter = :shell
   config.robot.log_level = :debug
+
+  config.default_inform_channel = "#workflow-pool"
+
+  config.projects = {
+    "chef" => {
+      pipeline: "chef-trigger-release",
+      github_url: "git@github.com:chef/chef.git",
+      version_bump_command: "bundle install && bundle exec rake version:bump && git checkout .bundle/config",
+      version_show_command: "bundle exec rake version:show",
+      dependency_update_command: "bundle install && bundle exec rake dependencies && git checkout .bundle/config",
+      inform_channel: "workflow-pool"
+    },
+    "chefdk" => {
+      pipeline: "chefdk-trigger-release",
+      github_url: "git@github.com:chef/chef-dk.git",
+      version_bump_command: "bundle install && bundle exec rake version:bump && git checkout .bundle/config",
+      version_show_command: "bundle exec rake version:show",
```

Comment:
It probably makes more sense to have a convention for what these scripts are called and then put them in the projects